docs/faq.md

```diff
@@ -86,6 +86,14 @@ Watchmaker is supported on RedHat 8, CentOS Stream 8, and Oracle Linux 8. See th
 
 Also: See CentOS Stream deprecation [notes](CentOS-Stream.md)
 
+## Does watchmaker support Enterprise Linux 9?
+
+Watchmaker is supported, to one degree or another, on all distros derived from
```

Comment:
begs the question, "to what degree"? i think just remove the qualification, it either is or isn't...

```suggestion
Watchmaker is supported on all distros derived from
```